e2e/libs/utility/globals.py

```diff
@@ -0,0 +1,37 @@
+import configparser
+
+from kubernetes import client
+
+CONFIG_FILE_PATH = 'settings.ini'
+
+def initialize_variables():
+    global RETRY_INTERVAL
+    global SSH_CONFIG_PATH
+    global CLOUD_PROVIDER
+    global K8S_DISTRO
+    global LONGHORN_CLIENT_URL
+    global K8S_API_CLIENT
+    global K8S_CR_API_CLIENT
+    global K8S_APP_API_CLIENT
+    global KUBECONFIG_PATH
+    
+    RETRY_INTERVAL = 5
+    SSH_CONFIG_PATH = '~/.ssh/config'
+    CLOUD_PROVIDER = 'aws' #aws
+    K8S_DISTRO = 'RKE2'
+    LONGHORN_CLIENT_URL = 'http://localhost:8080/'
+    K8S_API_CLIENT = client.CoreV1Api()
+    K8S_CR_API_CLIENT = client.CustomObjectsApi()
+    K8S_APP_API_CLIENT = client.AppsV1Api()
+    KUBECONFIG_PATH = ""
+    
```

Comment:
Why not use 'DEFAULT' config.sections()?

Reply:
Updated default values ​​taken from configuration files instead of assigned in source code.